### PR TITLE
fix: misc ui fixes

### PR DESCRIPTION
**Pull request by asdfryan**
Screenshot 2023-06-22 at 2 23 06 PM

### CODE REVIEW

File: apps/mgmt-ui/src/components/configuration/syncConfig/SyncConfigListPanel.tsx [hunk 1]
```diff
@@ -203,3 +205,20 @@ function millisToHumanReadable(millis: number): string {
 
   return humanReadable;
 }
+
+function getObjectsString(syncConfig: SyncConfig): string {
```
Comment (tomkit) [inline apps/mgmt-ui/src/components/configuration/syncConfig/SyncConfigListPanel.tsx:209]: we had an in-office discussion about separating the sync logs into {standard objects, other objects}. can we do that here too to be consistent from a ui perspective?

Comment (asdfryan) [reply apps/mgmt-ui/src/components/configuration/syncConfig/SyncConfigListPanel.tsx:209]: These aren't sync logs, this is just a short form list of objects that are being synced for the list view for sync configs.

**Review by tomkit — APPROVED** [review]
ideally we can have the columns be {standard objects, objects} so it matches the sync log information layout

File: apps/mgmt-ui/src/components/configuration/syncConfig/SyncConfigListPanel.tsx (hunk 1 above)
Comment (albertyfwu) [reply apps/mgmt-ui/src/components/configuration/syncConfig/SyncConfigListPanel.tsx:209]: i'm doing the "separating the sync logs into {standard objects, other objects}"

Comment (albertyfwu) [reply apps/mgmt-ui/src/components/configuration/syncConfig/SyncConfigListPanel.tsx:209]: https://github.com/supaglue-labs/supaglue/pull/1080

**Review by albertyfwu — APPROVED** [review]

**Pull request merged**